queries/subsquid/bsx/offerListByNftId.graphql

```diff
@@ -1,12 +1,12 @@
-query offerListByNftId($id: ID!) {
-  offers(where: {nft: {id_eq: $id}, status_eq: ACTIVE}) {
+query offerListByNftId($id: ID!, $account: String) {
+  offers(where: {nft: {id_eq: $id}, status_eq: ACTIVE, caller_not_eq: $account}) {
```

Comment:
I'm not sure if it works well on DestroyCollection Component. Could you please test it?
<img width="449" alt="image" src="https://user-images.githubusercontent.com/31397967/182007380-fc243e53-5166-4cd0-b744-c9f79213bb72.png">

Reply:
It can work since I use option variable rather than must

Reply:
> 

Besides, it's a different query in `DestroyCollection.vue` which is import from `@/queries/subsquid/general/collectionBurnableStats.graphql`.
The query I changed is `@/queries/subsquid/bsx/offerListByNftId.graphql`

Reply:
OKi